src/commands/mod.rs

```diff
@@ -569,12 +569,6 @@ impl DaemonControl {
                 bip32_derivation: change_desc.bip32_derivations(),
                 ..PsbtOut::default()
             });
-        } else if is_self_send {
```

Comment:
Would have been nice to have a comment above about how `change_amount` must be positive in case of a self send, and how if this doesn't hold the sanity checks will catch this.

Reply:
Ah yes, I'll do that.